cdrwtool/cdrwtool.c

```diff
@@ -416,11 +422,55 @@ int blank_disc(int fd, int type)
 		 return ret;
 	}
 
-	print_completion_info(fd);
-	return 0;
+	return wait_for_unit_ready(fd, "BLANK", WAIT_BLANK_SECONDS);
```

Comment:
I would suggest to pass timeout in hz units like in `wait_cmd` and `wait_cmd_sense` functions. Just to use parameters in these functions in the same unit. So it would not be suspicous that once variable `timeout` is in seconds and once in hz.